platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/snapshot/SignedStateFileUtils.java

```diff
@@ -32,6 +32,8 @@ public final class SignedStateFileUtils {
      */
     public static final String SIGNED_STATE_FILE_NAME = "SignedState.swh";
 
+    public static final String SIGNATURE_SET_FILE_NAME = "signatureSet.bin";
```

Comment:
separating the signatures will probably have some downstream effects on JRS, devops and so on. These need to be resolved before they can be separated

Reply:
Sure, I'll investigate it.